onnx/defs/nn/defs.cc

```diff
@@ -1416,4 +1402,67 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetDoc(LRN_ver1_doc)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+static const char* StringNormalizer_ver9_doc = R"DOC(
+[optional] Step1: Remove elements in X if they match any of the stop words so
```

Comment:
Here is an `optional Step 1` but I don't see a `Step 2`. Because I was involved in its original design, I can say its step 2 is `changing the case according to casechangeaction attribute. After reading this again, I feel it actually consists of some independent operators:
- Stop words remover.
- Lower, Upper
Do you think we should break it down?

Reply:
I think it's better to have the two covered in one op. Covering the two in one op does not add much complexity but help performance.